lib/rules/order-in-controllers.js

```diff
@@ -12,6 +12,7 @@ const ORDER = [
   'query-params',
   'inherited-property',
   'property',
+  'init',
```

Comment:
I'm somewhat confused by this... what about other inherited hook methods?

Reply:
I wanted to unify handling of controllers and routes with how it's done for components. But controllers don't have any hook methods. Also it seems to me that `init` is not really a hook, it looks more like a method dedicated to `EmberObject`.